build.gradle

```diff
@@ -1342,47 +1342,47 @@ project(':streams:upgrade-system-tests-21') {
   }
 }
 
-project(':jmh-benchmarks') {
-
-  apply plugin: 'com.github.johnrengelman.shadow'
-
-  shadowJar {
-    baseName = 'kafka-jmh-benchmarks-all'
-    classifier = null
-    version = null
-  }
-
-  dependencies {
-    compile project(':clients')
-    compile project(':streams')
-    compile libs.jmhCore
-    annotationProcessor libs.jmhGeneratorAnnProcess
-    compile libs.jmhCoreBenchmarks
-  }
-
-  jar {
-    manifest {
-      attributes "Main-Class": "org.openjdk.jmh.Main"
-    }
-  }
-
-
-  task jmh(type: JavaExec, dependsOn: [':jmh-benchmarks:clean', ':jmh-benchmarks:shadowJar']) {
-
-    main="-jar"
-
-    doFirst {
-      if (System.getProperty("jmhArgs")) {
-          args System.getProperty("jmhArgs").split(',')
-      }
-      args = [shadowJar.archivePath, *args]
-    }
-  }
-
-  javadoc {
-     enabled = false
-  }
-}
+//project(':jmh-benchmarks') {
+//
+//  apply plugin: 'com.github.johnrengelman.shadow'
+//
+//  shadowJar {
+//    baseName = 'kafka-jmh-benchmarks-all'
+//    classifier = null
+//    version = null
+//  }
+//
+//  dependencies {
+//    compile project(':clients')
+//    compile project(':streams')
+//    compile libs.jmhCore
+//    annotationProcessor libs.jmhGeneratorAnnProcess
+//    compile libs.jmhCoreBenchmarks
+//  }
+//
+//  jar {
+//    manifest {
+//      attributes "Main-Class": "org.openjdk.jmh.Main"
+//    }
+//  }
+//
+//
+//  task jmh(type: JavaExec, dependsOn: [':jmh-benchmarks:clean', ':jmh-benchmarks:shadowJar']) {
+//
+//    main="-jar"
+//
+//    doFirst {
+//      if (System.getProperty("jmhArgs")) {
+//          args System.getProperty("jmhArgs").split(',')
+//      }
+//      args = [shadowJar.archivePath, *args]
+//    }
+//  }
+//
+//  javadoc {
+//     enabled = false
+//  }
+//}
```

Comment:
Why this?

Reply:
Oh, this is my local build failure, will revert lol